### PR TITLE
docs: Add ad463x doc support

### PR DESCRIPTION
**Pull request by StancaPop**
Add documentation support for the AD463X testbench.

### CODE REVIEW

**Review by IstvanZsSzekely — CHANGES_REQUESTED** [review]

File: docs/testbenches/project_based/ad463x/index.rst [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+of the :git-hdl:`projects/ad4630_fmc` reference design.
+
+The entire HDL documentation can be found here
+:ref-hdl:`AD4630-FMC HDL project <ad4630_fmc>`.
+
+Block design
+-------------------------------------------------------------------------------
+
+The testbench block design includes part of the AD4630-FMC HDL reference design,
+along with VIPs used for clock & reset, DDR and PS AXI.
```
[the remark this answers is not in the document]
Comment (StancaPop) [reply docs/testbenches/project_based/ad463x/index.rst]: Updated.

File: docs/testbenches/project_based/ad463x/index.rst [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+Mode            1 Lane per channel 2 Lane per channel 4 lane per channel
+=============== ================== ================== ==================
+SPI mode        yes                yes                yes
+Echo Clock mode yes                yes                yes
+=============== ================== ================== ==================
+
+CPU/Memory interconnects addresses
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below are the CPU/Memory interconnects addresses used in this project:
```
[the remark this answers is not in the document]
Comment (StancaPop) [reply docs/testbenches/project_based/ad463x/index.rst]: Done.

File: docs/testbenches/project_based/ad463x/index.rst [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+   positive edge +20ns until the second next consecutive CNV positive edge -20ns
+-  DDR_EN: defines the type of data transfer. In echo and master clock mode
+   the SDI lines can have Single or Double Data Rates.
+   Options: 0 - MISO runs on SDR, 1 - MISO runs on DDR.
+
+Build parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+ 
+The parameters mentioned above can be configured during the build, like in the
+following example:
```
[the remark this answers is not in the document]
Comment (StancaPop) [reply docs/testbenches/project_based/ad463x/index.rst]: Updated.

File: docs/testbenches/project_based/ad463x/index.rst [hunk 4]
```diff
@@ -0,0 +1,368 @@
...
+-------------------------------------------------------------------------------
+
+The first steps of this testbench are:
+
+* Create the environment
+* Start the environment
+* Start the clocks
+* Assert the resets
+
+followed by custom tests (Sanity, FIFO SPI and Offload SPI tests).
```
[the remark this answers is not in the document]
Comment (StancaPop) [reply docs/testbenches/project_based/ad463x/index.rst]: Done.

**Review by IstvanZsSzekely — APPROVED** [review]

**Pull request merged**